app/models/task.py

```diff
@@ -1,5 +1,27 @@
-from app import db
+from app import db 
```

Comment:
Huh, extra space here. It doesn't bother anything in this case as well as most cases, though, I just am nitpicky.